client/src/App.tsx

```diff
@@ -2067,6 +2129,17 @@ function App() {
 			onClick={onClickApp}
 		>
 			<MetamaskSection />
+
+			<Route path="/settings">
+				<SettingsPanel 
```

Comment:
Good use of route, I think ideally this logic (onSubmitUrl, onChangeName, etc) would be handled in the settings panel with the socket thru context, the less logic we have in App the better

Reply:
so do i create a new context like maps and move everything to that?

Reply:
or is this a note for v2?